covid_tracing/src/main.rs

```diff
@@ -116,37 +127,49 @@ fn main() {
     df.add_edge(notifs_out, notifs_in);
 
     let reactor = df.reactor();
-    let (is_done, set_done) = std::sync::mpsc::sync_channel(0);
+
+    let all_people = people::get_people();
+    for person in &all_people {
+        people_send.send(person.clone()).unwrap();
+        reactor.trigger(people_op_id).unwrap();
+    }
```

Comment:
Can put this in another `std::thread::spawn` to emphasize the asynchrony?

Reply:
Done.